src/Youtube.js

```diff
@@ -617,6 +617,12 @@ THE SOFTWARE. */
     preload: function() {},
     load: function() {},
     reset: function() {},
+    networkState: function () {
+      return 0;
```

Comment:
should we update these values to something reasonable once the video has loaded?